Makefile

```diff
@@ -23,7 +23,11 @@ run-graphql-demo: build ## run the demo example
 run-graphcool-demo: build ## run the demo example
 	@cd examples/graphcool-demo && yarn start
 
-build-react-admin:
+build-react-admin-core:
+	@rm -rf ./packages/react-admin-core/lib
+	@NODE_ENV=production ./node_modules/.bin/babel ./packages/react-admin-core/src -d ./packages/react-admin-core/lib --ignore spec.js,test.js
+
+build-react-admin: build-react-admin-core
```

Comment:
I'd prefer that you add the `build-react-admin-core` as a predecessor to the `build` target, like all the other packages.